src/components/home/questionnaire-progress.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Typography, CircularProgress } from "@mui/material";
+import { useEffect, useState } from "react";
+import { useLambdasApi } from "src/hooks/use-api";
+import { useAtomValue } from "jotai";
+import { userProfileAtom } from "src/atoms/auth";
+import { usersAtom } from "src/atoms/user";
+import type { Questionnaire, User } from "src/generated/homeLambdasClient";
+import strings from "src/localization/strings";
+
+/**
+ * Component to display user's questionnaire progress on home screen
+ * Located in home folder as it's specific to the dashboard
+ */
+const QuestionnaireProgress = () => {
+  const { questionnairesApi } = useLambdasApi();
+  const userProfile = useAtomValue(userProfileAtom);
+  const users = useAtomValue(usersAtom);
+  const [questionnaires, setQuestionnaires] = useState<Questionnaire[]>([]);
+  const [loading, setLoading] = useState(true);
```

Comment:
We initialize loading as false and only set it to true when we need to  (when making  request).

Reply:
So this should be set to false